test/helpers/common.js

```diff
@@ -960,3 +977,5 @@ class DataTransferObject {
     this.data[type] = value;
   }
 }
+
+export { getRenderedBorderPaths, getRenderedBorderPathExistence, getRenderedBorderStyles } from '../../src/3rdparty/walkontable/test/helpers/common';
```

Comment:
I think that it is a good practice to put all sorts of imports or re-exports on the top of the file.